frontend/src/metabase/query_builder/selectors.js

```diff
@@ -298,6 +299,7 @@ export const getSampleDatabaseId = createSelector(
     const sampleDatabase = _.findWhere(databases, { is_sample: true });
     return sampleDatabase && sampleDatabase.id;
   },
+  { devModeChecks: { inputStabilityCheck: "never" } },
```

Comment:
:x: This warning is correct and can be fixed by returning the same empty array in line 265.